content/tutorials/esc-external-secret-operator/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: "Integrate with External Secrets Operator (ESO)"
+title_tag: Integrate with External Secrets Operator (ESO) | Learn Pulumi
+h1: "Integrate with External Secrets Operator (ESO)"
+layout: single
+description: |
+  Pulumi ESC integrates with the External Secrets Operator (ESO) to manage and deliver secrets in Kubernetes clusters.
+meta_desc: Pulumi ESC integrates with the External Secrets Operator (ESO) to manage and deliver secrets in Kubernetes clusters.
+weight: 999
+meta_image: meta.png
+summary: |
+  [External Secrets Operator](https://external-secrets.io/latest/) is a Kubernetes operator that integrates external secret management systems with Kubernetes. By using External Secrets Operator, you have several advantages over Kubernetes native secrets:
+
+  - Sensitive data is stored and managed by an external service outside the Kubernetes cluster and this leads to better security and compliance.
+  - External Secrets Operator supports multiple different sources, so you can use the same operator to manage secrets and configuration from different sources.
+  - Take advantage of the advanced features of the secret provider, such as encryption of data at rest and scenarios like secret rotation.
+
+  Since version `0.10.0` External Secrets Operator supports Pulumi ESC as a secret provider.
+youll_learn:
+- How to deploy the External Secrets Operator using Helm or Pulumi
+- How to manage secrets stored in Pulumi ESC using External Secrets Operator
+- How to push secrets from Kubernetes to Pulumi ESC
+estimated_time: 15
+collections:
+- pulumi-esc
+- kubernetes
```

Comment:
For now, I would leave this collection off. The reason being, if we merge as-as that creates a new "Browse by Cloud" left nav category, with only a single item under it "Kubernetes" which itself only has a single tutorial under it (this one). That will end up being confusing to the reader, because we have so many tutorials for other clouds that are not yet tagged in that way.

That said, I like the idea of leveraging the "Browse by cloud" navigation, but I think we should do that in a separate PR that organizes all the existing tutorials by cloud in a single changeset, so that it's fully populated when it goes live.

Reply:
<img width="738" alt="image" src="https://github.com/user-attachments/assets/cb289f35-8c6e-4a3c-a4c9-b765d6d6a185" />